### PR TITLE
Don't mark CE methods on builders as unused

### PR DESCRIPTION
**Pull request by cartermp**
Before:

image

After:

image

Probably need to figure out the `BindN` ones too but I figured I'd get this up into a PR first

### CODE REVIEW

**Comment by cartermp** [comment]
aw hamburgers I need to do ugly stuff instead of using our reverse indexing

File: src/fsharp/symbols/Symbols.fs [hunk 1]
```diff
@@ -2019,6 +2019,35 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
...
+    member x.IsComputationExpressionMethod =
+        let isNameN (methodName: string) (methodKind: string) =
+            if methodName.StartsWith(methodKind) then
+                let (didParse, _) = System.Int32.TryParse(methodName.[methodKind.Length..])
+                didParse
+            else
+                false
+        let isBindNReturn (methodName: string) =
+            if methodName.StartsWith("Bind") && methodName.EndsWith("Return") then
+                let (didParse, _) = System.Int32.TryParse(methodName.[4..methodName.Length-7]) // replace this disgusting mess with reverse indexing some time
```
Comment (brettfo) [inline src/fsharp/symbols/Symbols.fs:2031]: I'd like to see a test, particularly for this monstrosity.

Comment (cartermp) [reply src/fsharp/symbols/Symbols.fs:2031]: I hate so much the giant baby

Comment (TIHan) [reply src/fsharp/symbols/Symbols.fs:2031]: Ouch. There no alternative way around this?

Comment (cartermp) [reply src/fsharp/symbols/Symbols.fs:2031]: Gotta find the number between `Bind` and `Return` in `Bind123Return`, so yeah I think it is. I guess I could pull out some values into names to help

**Pull request closed without merging**